src/hooks/useAsset.js

```diff
@@ -28,9 +73,17 @@ export default function useAsset(asset) {
             allAssets,
             matchesProperty('address', asset.mainnet_address || asset.address)
           );
+
+      if (!matched) {
+        matched = find(allAssets, matchesProperty('uniqueId', asset.uniqueId));
+      }
       if (!matched) {
         matched = genericAssets?.[asset.mainnet_address || asset.address];
       }
+
+      if (!matched && asset.uniqueId === 'eth') {
```

Comment:
btw what is this for? why do we want to use a default 0 eth fallback

Reply:
![image](https://user-images.githubusercontent.com/25709300/145950658-e6c4f0ed-aa8c-4f55-a6b2-4c6769ea004e.png)